terratag_test.go

```diff
@@ -95,6 +95,7 @@ func TestTerragruntWithCache(t *testing.T) {
 	itShouldRunTerratagTerragruntMode(out, g)
 	itShouldRunTerragruntValidate(out, g)
 	itShouldGenerateExpectedTerragruntTerratagFiles(entryDir, g)
+	itShouldRunTerratagTerragruntModeDryRun(out, g)
```

Comment:
updating the tests to make run a dry run at the end and verify it's resolved.
I'm not running a dry run at the beginning because some tests are not expected to be tagged.